RecoHI/Configuration/python/Reconstruction_HI_cff.py

```diff
@@ -35,6 +35,20 @@
 
 from RecoLocalCalo.Configuration.hcalGlobalReco_cff import *
 
+#post PF egamma stuff
+from RecoHI.HiEgammaAlgos.HiEgammaPostPF_cff import *
+
+from RecoHI.HiJetAlgos.HiRecoPFJets_cff import *
+
+#reduced rechits
+from RecoEcal.EgammaClusterProducers.reducedRecHitsSequence_cff import *
+from RecoEcal.EgammaCoreTools.EcalNextToDeadChannelESProducer_cff import *
```

Comment:
Presumably, the same reason it's needed here:
http://cmslxr.fnal.gov/source/RecoEcal/Configuration/python/RecoEcal_cff.py
which is to avoid the following:

Exception Message:
No "EcalNextToDeadChannelRcd" record found in the EventSetup for synchronization value
Run: 1 LuminosityBlock: 1 Event: 0 Time: 1
 Please add an ESSource or ESProducer that delivers such a record.
----- End Fatal Exception --